src/components/shared/ConfirmDialog.tsx

```diff
@@ -24,22 +25,40 @@ const ConfirmDialog = () => {
       aria-describedby={descriptionId}
       isOpen={isOpen}
       focusAfterCloseRef={openConfirmationDialogButtonRef}
+      variant={dialogType === 'deleteProject' ? 'danger' : 'primary'}
     >
       <Dialog.Header
         id={titleId}
         title={title}
-        iconLeft={<IconQuestionCircle aria-hidden="true" />}
+        iconLeft={
+          dialogType === 'deleteProject' ? (
+            <IconAlertCircle aria-hidden="true" />
+          ) : (
+            <IconQuestionCircle aria-hidden="true" />
+          )
+        }
       />
       <Dialog.Content>
         <p id={descriptionId} className="text-body">
           {description}
         </p>
       </Dialog.Content>
       <Dialog.ActionButtons>
-        <Button onClick={proceed as (value: unknown) => void}>{t('proceed')}</Button>
-        <Button onClick={cancel as (value: unknown) => void} variant="secondary">
+        <Button
+          onClick={cancel as (value: unknown) => void}
+          variant="secondary"
+          theme={dialogType === 'deleteProject' ? 'black' : 'default'}
+        >
           {t('cancel')}
         </Button>
+        <Button
+          data-testid={dialogType === 'deleteProject' ? 'delete-project-dialog-button' : ''}
+          onClick={proceed as (value: unknown) => void}
+          variant={dialogType === 'deleteProject' ? 'danger' : 'primary'}
+          iconLeft={dialogType === 'deleteProject' ? <IconTrash aria-hidden="true" /> : ''}
```

Comment:
Icon and variant should be memoized:

  const confirmButtonVariant = useMemo(() => ...)
  const confirmButtonIcon = useMemo(() => ...)